doc/en/reference.rst

```diff
@@ -27,6 +27,8 @@ pytest.skip
 
 .. autofunction:: _pytest.outcomes.skip(msg, [allow_module_level=False])
 
+.. _`pytest.importorskip ref`:
```

Comment:
```suggestion
.. _`pytest.importorskip`:
```

To fix:

> Warning, treated as error:
/home/travis/build/pytest-dev/pytest/doc/en/skipping.rst:182:undefined label: pytest.importorskip (if the link has no caption the label must precede a section header)

That's why I had a "?" with the suggestion.. ;)